gluten-core/src/main/java/org/apache/gluten/jni/JniLibLoader.java

```diff
@@ -50,14 +47,6 @@ public class JniLibLoader {
   private static final Set<String> REQUIRE_UNLOAD_LIBRARY_PATHS =
       Collections.synchronizedSet(new LinkedHashSet<>());
 
-  static {
-    SparkShutdownManagerUtil.addHookForLibUnloading(
-        () -> {
-          forceUnloadAll();
```

Comment:
It seems the `forceUnloadAll` is used to fix [the coredump in ch backends](https://github.com/apache/incubator-gluten/pull/1143). Can we remove this now? @taiyang-li @baibaichen

Reply:
Yes, it is still needed.

Reply:
It seems can not be removed, or will still coredump